avocado_vt/test.py

```diff
@@ -159,7 +159,7 @@ def runTest(self):
         # This trick will give better reporting of virt tests being executed
         # into avocado (skips, warns and errors will display correctly)
         except error.TestNAError, details:
```

Comment:
OK, first like kept as is, as it comes from autotest as pointed out by @will-Do .